src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -186,15 +195,18 @@ public void set(Object text) {
      */
     public void selectDropdownMenu(Class type) {
         click();
-        findCaption(type,findDropDownMenuItem).click();
-        elasticSleep(1000);
```

Comment:
sleeps must die :)

now handled in [`findDropDownMenuItem`](https://github.com/jenkinsci/acceptance-test-harness/pull/619/files#diff-924ccb08900e51e37d8f6768a863e87795544b16c56986caa3047a709fb7d0c5R233-R238)